src/app/f2f/controllers/photoIdSelectionThinFile.js

```diff
@@ -23,7 +23,6 @@ class PhotoIdSelectionThinFileController extends BaseController {
             { req, res }
           );
           req.sessionModel.set(APP.PHOTO_ID_OPTIONS.UK_PASSPORT, true);
-          req.sessionModel.set("selectedDocument", "UK passport");
```

Comment:
It looks like there is still a test for this, can we remove it?

Reply:
Of course

Reply:
There isn't a test for this value but there is a variable called selectedDocumentValue when testing for the photoIdChoice, is that what you mean? I can rename this for clarity

Reply:
yes please 👍